docs/qcfractal/source/install.rst

```diff
@@ -34,42 +39,57 @@ If you want to use a different name than the environment file, you can add a ``-
 
 The environments must be installed as new environments and cannot be installed into existing ones.
 
+
 Pip
 ---
 
-qcfractal may be installed with ``pip``::
+.. warning::
+
+   Installing Fractal from PyPi/``pip`` requires an existing PostgreSQL installation on your computer. Whether that be
+   through a native install on your device (e.g. managed clusters), a direct installer, ``yum`` install, a ``conda``
+   install, or otherwise; it must be installed first or the ``Psycopg2`` package will complain about missing the
+   ``pg_config``. Installation of PostgreSQL manually is beyond the scope of these instructions, so we recommend
+   either using a :ref:`Conda install of Fractal <conda-install>` or contacting your systems administrator.
+
+If you have PosgreSQL installed already, you can also install QCFractal using ``pip``:
+
+.. code-block:: console
+
+   >>> pip install qcfractal
+
+
+Test the Installation
+---------------------
+
+.. note::
+
+   There are several optional packages Fractal can interface with for additional features such as visualization,
+   :term:`Queue Adapters <Queue Adapter>`, and services. These are not installed by default and so you can expect many of the tests
+   will be marked with ``skip`` or ``s``.
 
-    pip install qcfractal
+You can test to make sure that Fractal is installed correctly by first installing ``pytest``.
 
-Install from Source
--------------------
+From ``conda``:
 
-To install qcfractal from source, clone the repository from `github
-<https://github.com/molssi/qcfractal>`_::
+.. code-block:: console
 
-    git clone https://github.com/molssi/qcfractal.git
-    cd qcfractal
-    python setup.py install
+   >>> conda install pytest -c conda-forge
 
-or use ``pip`` for a local install::
+From ``pip``:
 
-    pip install -e .
+.. code-block:: console
 
-It is recommended to setup a testing environment using ``conda``. This can be accomplished by::
+   >>> pip install pytest
 
-    cd qcfractal
-    python devtools/scripts/conda_env.py -n=qcf_test -p=3.7 devtools/conda-envs/openff.yaml
-    conda activate qcarchive
-    pip install -e .
+Then, run the following command:
 
-This installs all the dependencies to setup a production background in a new conda environment,
-activate the environment, and then install Fractal into development mode.
+.. code-block::
 
-Test
-----
+   >>> pytest -p qcfractal.testing --pyargs qcfractal
```

Comment:
Cool does this work while in another directory with the `-p` option?

Reply:
Yes. Since the `testing.py` file is importable from `qcfractal`, it does a python import as a plugin and therefore finds the missing fixtures it needs. I think we technically loose access to the `--runslow`` and ``--runexamples`` flags though. But thats an acceptable loss for now.

Reply:
Gotcha. I think the files it is complaining about is where we forgot to do an explicit `import`. So we might be able to skip this in the future. `--runslow` would be good to keep however since the slow tests make testing a bit tedious.

Reply:
If we want to keep the `--runslow`, we have to ship around the `conftest.py` file which sits at the same level as `setup.py` and not packaged with the code right now.

Reply:
If we package it in the egg will it still work?

Reply:
Not sure. What would happen if we move the `conftest.py` file down one level into the `qcfractal` folder so it is shipped around with the package by default? We dont add it to any of the `__init__.py` files, but I think that would allow pytest to just find it and work if I am following correctly?